crates/router/src/connector/worldpay/requests.rs

```diff
@@ -6,15 +6,15 @@ pub struct BillingAddress {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub city: Option<String>,
     #[serde(skip_serializing_if = "Option::is_none")]
-    pub address2: Option<String>,
-    pub postal_code: String,
+    pub address2: Option<Secret<String>>,
+    pub postal_code: Secret<String>,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub state: Option<String>,
     #[serde(skip_serializing_if = "Option::is_none")]
-    pub address3: Option<String>,
+    pub address3: Option<Secret<String>>,
     pub country_code: String,
```

Comment:
`country_code`'s type can be changed from `String` to our `Currency` enum.

Accordingly update at all the places if necessary.

Reply:
Field `country_code` under Submerchant, is expected to have a numeric value according to the [connector docs](https://developer.worldpay.com/docs/access-worldpay/api/references/payments/authorizations). Hence not updating the type of this field.